netlink.go

```diff
@@ -351,6 +351,17 @@ func assembleService(attrs []syscall.NetlinkRouteAttr) (*Service, error) {
 
 	}
 
+	// in older kernels (< 3.18), the svc address family attribute may not exist so we must
```

Comment:
AFAICT this is only an issue for destination addresses since `IPVS_DEST_ATTR_ADDR_FAMILY` was added later while `IPVS_SVC_ATTR_AF` had always existed, but this extra check doesn't seem to hurt.